### ISSUE TITLE
Allow repairing and removing a switch if the container was accidentally removed

### ISSUE THREAD

**Issue by NoamDev** [OCamlPro/freeton_wallet#69]
right now, there is no command to recreate a docker container if you accidentally removed it. (maybe it should be auto created?).
also, if the  container was removed, you cannot remove the switch, since it fails in docker container removal.

**Comment by lefessan** [comment]
Indeed, there is no command for the first one. Note that you can still recreate another switch with the same accounts using `ft account copy from OLD_SWITCH`. Anyway, if you removed the container, their states were removed.

I think you can use `ft switch remove SWITCH -f`, i.e. use `-f` to force container removal after docker failure
